Migration.Toolkit.CLI/Migration.Toolkit.CLI.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>net6.0</TargetFramework>
```

Comment:
Maybe we go to `net8.0` now?

It's probably best to set this in a solution-wide `Directory.Build.props`